source/uwp/Renderer/idl/AdaptiveCards.XamlCardRenderer.idl

```diff
@@ -1444,5 +1450,56 @@ namespace AdaptiveCards
         {
             [default] interface IXamlCardGetResourceStreamArgs;
         };
+
+        [
+            uuid(7bf9ceac-105c-421a-8b4e-8e5be9ead986),
+            version(NTDDI_WIN10_RS1),
+            exclusiveto(RenderedAdaptiveCard)
+        ]
+        interface IRenderedAdaptiveCard : IInspectable
+        {
+            [propget] HRESULT OriginatingCard([out, retval] AdaptiveCard** value);
+
+            [propget] HRESULT FrameworkElement([out, retval] Windows.UI.Xaml.UIElement** value);
+
+            [propget] HRESULT Errors([out, retval] Windows.Foundation.Collections.IVector<HSTRING>** value);
+
+            [propget] HRESULT Warnings([out, retval] Windows.Foundation.Collections.IVector<HSTRING>** value);
+
+            [propget] HRESULT IsRenderedSuccessfully([out, retval] boolean* value);
+
+            [propget] HRESULT UserInputs([out, retval] AdaptiveInputs** value);
+
+            [eventadd] HRESULT Action([in] Windows.Foundation.TypedEventHandler<RenderedAdaptiveCard*, AdaptiveActionEventArgs*>* pHandler, [out][retval] EventRegistrationToken* pToken);
+            [eventremove] HRESULT Action([in] EventRegistrationToken token);
+        };
+
+        [
+            version(NTDDI_WIN10_RS1),
+            activatable(NTDDI_WIN10_RS1)
+        ]
+        runtimeclass RenderedAdaptiveCard
+        {
+            [default] interface IRenderedAdaptiveCard;
+        };
+
+        [
+            uuid(a172be96-7ab5-4faa-a0fd-99e8160e359e),
+            version(NTDDI_WIN10_RS1),
+            exclusiveto(AdaptiveInputs)
+        ]
+        interface IAdaptiveInputs : IInspectable
+        {
+            [propget] HRESULT AsJson([out, retval] Windows.Data.Json.JsonObject** value);
```

Comment:
>AsJson [](start = 30, length = 6)

Should this take InputValueMode as a parameter?

Reply:
Oops... Yeah, that was the idea. That's why I introduced that enum in the first place even though it only has one value right now.

---
In reply to: [140818421](https://github.com/Microsoft/AdaptiveCards/pull/678#discussion_r140818421) [](ancestors = 140818421)